lib/ntbb-session.lib.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
 require_once dirname(__FILE__) . '/ntbb-database.lib.php';
```

Comment:
tbh this is all really old code that I wrote because I didn't understand include paths. These can all be rewritten:

```suggestion
require_once './ntbb-database.lib.php';
```